libres/lib/enkf/local_config.cpp

```diff
@@ -247,3 +229,33 @@ local_config_get_updatestep(const local_config_type *local_config) {
 
     return updatestep;
 }
+
+namespace {
+
+LocalObsData &get_obsdata(py::handle obj, const std::string &obs_key) {
+    auto *local_config = reinterpret_cast<local_config_type *>(
+        PyLong_AsVoidPtr(obj.attr("_BaseCClass__c_pointer").ptr()));
+    auto *obsdata = local_config_get_obsdata(local_config, obs_key.c_str());
+    return *obsdata;
+}
+
+LocalObsData copy_obsdata(py::handle obj, const std::string &obs_key,
+                          const std::string &target_key) {
+    auto *local_config = reinterpret_cast<local_config_type *>(
+        PyLong_AsVoidPtr(obj.attr("_BaseCClass__c_pointer").ptr()));
```

Comment:
```suggestion
    auto *local_config = ert::from_cwrap<local_config_type>(obj);
```

Reply:
Fixed in https://github.com/equinor/ert/pull/2966.